public/lib/angular-bootstrap/css/carousel.less

```diff
@@ -390,7 +390,7 @@ hr {
   opacity: 0.5;
   filter: alpha(opacity=50);
   font-size: 20px;
-  color: #ffffff;
+  color: #464646;
```

Comment:
Avoid modifying third-party/generated code. Instead you could override the style.